packages/airnode-deployer/src/handlers/gcp/index.ts

```diff
@@ -44,7 +44,6 @@ async function initializeProvider(payload: InitializeProviderPayload, res: Respo
   const [err, initializedState] = await utils.go(() => handlers.initializeProvider(stateWithConfig));
   if (err || !initializedState) {
     const msg = `Failed to initialize provider: ${stateWithConfig.settings.name}`;
-    console.log(err!.toString());
```

Comment:
Removing this is changing behaviour. Maybe it should be added back?

Reply:
Did you? 👀

Reply:
ping

Reply:
Yes, sorry, it'll appear in the next commit (hence why I haven't marked this as resolved yet).

Reply:
It has been replaced with `logger.log`.